ecommers/lib/ui/pages/authorization/authorization_page.dart

```diff
@@ -24,10 +24,13 @@ class AuthorizationPage extends StatelessWidget {
       providers: [
         ChangeNotifierProvider(create: (_) => LogInProviderModel(context)),
         ChangeNotifierProvider(create: (_) => SignUpProviderModel(context)),
-        ChangeNotifierProxyProvider2<SignUpProviderModel, LogInProviderModel,
-            BusyProviderModel>(
+        ChangeNotifierProvider(
+            create: (_) => ForgotPasswordProviderModel(context)),
+        ChangeNotifierProxyProvider3<SignUpProviderModel, LogInProviderModel,
```

Comment:
ChangeNotifierProxyProvider is used here to show Loading. As I see ForgotPasswordProviderModel doesn't change isBusy property.
So, you don't need to change ChangeNotifierProxyProvider2 to ChangeNotifierProxyProvider3